main.go

```diff
@@ -41,6 +45,10 @@ import (
 	//+kubebuilder:scaffold:imports
 )
 
+const (
+	apiPort = 3006
```

Comment:
I think it would be better to assign the port in a values.yaml file and then read environment variable holding the port number.